ci_framework/roles/ci_nmstate/tasks/nmstate_k8s_install.yml

```diff
@@ -0,0 +1,106 @@
+  ansible.builtin.copy:
+    dest: "{{ cifmw_ci_nmstate_manifests_dir }}/nmstate-{{ item.kind | lower }}-olm.yaml"
+    content: "{{ item | to_nice_yaml }}"
+  loop:
+    - "{{ cifmw_ci_nmstate_olm_operator_group }}"
+    - "{{ cifmw_ci_nmstate_olm_subscription }}"
+  loop_control:
+    label: "{{ item.metadata.name }}"
+
+- name: Create the OLM subscription resources
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    definition: "{{ item }}"
+    state: present
+  loop:
+    - "{{ cifmw_ci_nmstate_olm_operator_group }}"
+    - "{{ cifmw_ci_nmstate_olm_subscription }}"
+  loop_control:
+    label: "{{ item.metadata.name }}"
+
+- name: Wait for nmstate operator to be intalled
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
+    cmd: >-
+      oc get ClusterServiceVersion
+      -n "{{ cifmw_ci_nmstate_namespace }}"
+      -o jsonpath='{.items[*].status.phase}'
+  changed_when: false
+  register: _cifmw_ci_nsmate_csv_out
+  retries: 12
+  delay: 10
+  until:
+    - _cifmw_ci_nsmate_csv_out is defined
+    - _cifmw_ci_nsmate_csv_out.failed is false
+    - _cifmw_ci_nsmate_csv_out.stdout_lines | length > 0
+    - "(_cifmw_ci_nsmate_csv_out.stdout_lines[0] | lower) == 'succeeded'"
+
+- name: Save nmstate operator config as an artifact
+  ansible.builtin.copy:
+    dest: "{{ cifmw_ci_nmstate_manifests_dir }}/nmstate-{{ cifmw_ci_nmstate_operator_config.kind | lower }}-olm.yaml"
+    content: "{{ cifmw_ci_nmstate_operator_config | to_nice_yaml }}"
+
+- name: Trigger nmstate operator to install nmstate
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    definition: "{{ cifmw_ci_nmstate_operator_config }}"
+    state: present
+
+- name: Wait for nsmtate Deployments to be ready
+  kubernetes.core.k8s_info:
+    kind: Deployment
+    label_selectors:
+      - "app=kubernetes-nmstate"
+    namespace: "{{ cifmw_ci_nmstate_namespace }}"
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+    wait: true
+    wait_sleep: 10
+    wait_timeout: 600
+    wait_condition:
+      type: Available
+      status: "True"
+
+- name: Wait for nsmtate DaemonSet to be ready
+  kubernetes.core.k8s_info:
+    kind: DaemonSet
+    label_selectors:
+      - "app=kubernetes-nmstate"
+    namespace: "{{ cifmw_ci_nmstate_namespace }}"
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+  register: _cifmw_ci_nsmate_daemonset_out
```

Comment:
since that var is used only for the `until`, why can't we get a shorter name - for instance, removing the `cifmw_ci` prefix?